special-pages/pages/history/app/global/Providers/SelectionProvider.js

```diff
@@ -99,11 +99,7 @@ export function useRowInteractions(mainRef) {
             if (handled) {
                 event.preventDefault();
                 event.stopImmediatePropagation();
-            } else {
-                console.log('did not handle selection');
             }
-        } else {
-            dispatch({ kind: 'reset', reason: 'click occurred outside of rows' });
```

Comment:
this is where it reset previously, but this handler was only scoped to the `main` tag.